General brush up of PowerToys docs (#4558)
MicrosoftDocs/windows-uwp: hub/powertoys/fancyzones.md

# FancyZones utility

> FancyZones is a window manager utility for arranging and snapping windows into efficient layouts to improve your workflow and restore layouts quickly. FancyZones allows you to define a set of zone locations to be used as targets for windows on your desktop. When you drag a window into a zone, or enter the associated keyboard shortcut, the window is resized and repositioned to fill that zone.

Reviewer: ```suggestion
FancyZones is a window manager utility for arranging and snapping windows into efficient layouts to improve your workflow and restore layouts quickly. You can define a set of zone locations to use as targets for windows on your desktop. When you drag a window into a zone, or enter the associated keyboard shortcut, the window is resized and repositioned to fill that zone.
```

## Snapping to a single zone with mouse

> First, drag the window. By default, you will also need to press and hold the <kbd>Shift</kbd> key (This setting can be disabled via Settings). You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone.

Reviewer: ```suggestion
Drag the window. By default, you'll also need to select and hold the <kbd>Shift</kbd> key. You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone.
```

I don't think you need to specify that the activation key can be disabled at this point because it is implied with 'By default'.

'Select' is preferred to 'press' when [describing keyboards](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts).

You can also trigger zone selection mode by using a non-primary mouse button if "Use non-primary mouse button to toggle zone activation" is selected.

> If both "Hold Shift key to activate zones while dragging" and "Use non-primary mouse button to toggle zone activation" are cleared, zones will appear immediately after you start dragging the window.

Reviewer: ```suggestion
If both **Hold Shift key to activate zones while dragging** and **Use non-primary mouse button to toggle zone activation** are cleared, zones will appear immediately after you start dragging the window.
```

Bold setting names

![FancyZones in action screenshot.](../images/pt-fancy-zones2.png)

## Snapping to a single zone with keyboard

> First select **Override Windows Snap** in the FancyZones settings. Use <kbd>Win</kbd>+[arrow keys] to snap a window to a zone. Next zone to snap can be chosen either by its index or by relative position.

Reviewer: ```suggestion
Select **Override Windows Snap** in the FancyZones settings. Use <kbd>Win</kbd>+[arrow keys] to snap a window to a zone. Use **Move windows based on** to choose whether to move windows based the zone index or a window's relative position.
```

![Settings for Snapping to Multiple Zones via Keyboard.](../images/pt-window-snap-multiple-zones-w-keyboard-settings.png)

## Snapping to multiple zones

A window can be snapped to more than one zone in the following ways.

### Snapping to two zones by hovering the edges

> If two zones are adjacent, a window can be snapped to the sum of their area (rounded to the minimum rectangle that contains both). When the mouse cursor is near the common edge of two zones, both zones are activated simultaneously, allowing you to drop the window into both zones.

Reviewer: ```suggestion
If two zones are adjacent, you can snap a window to the sum of their area (rounded to the minimum rectangle that contains both). When the mouse cursor is near the common edge of two zones, both zones are activated simultaneously, allowing you to drop the window into both zones.
```

Passive voice

> ### Snapping to multiple zones with the mouse + keyboard

Reviewer: ```suggestion
### Snapping to multiple zones with the mouse and keyboard
```

Drag the window until one zone is activated, then press and hold the <kbd>Control</kbd> key while dragging the window to select multiple zones.

![Two Zones Activation screenshot.](../images/pt-fancyzones-twozones.png)

### Snapping to multiple zones with only the keyboard

> First select **Override Windows Snap** in the FancyZones settings and select **Move windows based on: Relative position**. Use <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+[arrow keys] to expand the window to multiple zones.

Reviewer: ```suggestion
Select **Override Windows Snap** in the FancyZones settings and select **Move windows based on: Relative position**. Use <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+[arrow keys] to expand the window to multiple zones.
```

| <kbd>⊞ Win</kbd>+<kbd>Left/Right</kbd> | Move focused window between zones (only if **Override Windows Snap hotkeys** is selected and **Zone index** is chosen; in that case only the <kbd>⊞ Win</kbd>+<kbd>Left</kbd> and <kbd>⊞ Win</kbd>+<kbd>Right</kbd> are overridden, while the <kbd>⊞ Win</kbd>+<kbd>Up</kbd> and <kbd>⊞ Win</kbd>+<kbd>Down</kbd> keep working as usual) |
| <kbd>⊞ Win</kbd>+<kbd>Left/Right/Up/Down</kbd> | Move focused window between zones (only if **Override Windows Snap hotkeys** is selected and **Relative position** is chosen; in that case all the <kbd>⊞ Win</kbd>+<kbd>Left</kbd>, <kbd>⊞ Win</kbd>+<kbd>Right</kbd>, <kbd>⊞ Win</kbd>+<kbd>Up</kbd> and <kbd>⊞ Win</kbd>+<kbd>Down</kbd> are overridden) |
| <kbd>⊞ Win</kbd>+<kbd>PgUp/PgDn</kbd> | Cycle between windows snapped to the same zone |
| <kbd>⊞ Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>[number]</kbd> | Quickly apply custom layout (you need to assign number to the custom layout in the editor first) |

FancyZones doesn't override the Windows <kbd>⊞ Win</kbd>+<kbd>Shift</kbd>+[arrow keys] to quickly move a window to an adjacent monitor.

### Snapping apps with elevated permission

> To snap applications that are elevated (such as Windows Terminal or Task Manager) run PowerToys in administrator mode. Read [Running as administrator](administrator.md) for more information.

Reviewer: ```suggestion
To snap applications that are elevated (such as Windows Terminal or Task Manager), run PowerToys in administrator mode. Read [Running as administrator](administrator.md) for more information.
```

Missing comma

## Getting started with the editor

FancyZones includes an editor to give you more control over your window layouts that can be accessed in the PowerToys Settings.

### Launch zones editor

> Launch the zones editor using the button in the PowerToys Settings tab or by pressing <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>`</kbd> ("back-tick" or "accent grave"; note that this shortcut can be changed in the Settings tab).

Reviewer: ```suggestion
Launch the zones editor by selecting **Launch layout editor** in the PowerToys Settings tab or by pressing <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>`</kbd> ("back-tick" or "accent grave"). You can change the FancyZones layout editor shortcut in PowerToys Settings.
```

Simplified sentence structure.

![FancyZones Settings UI.](../images/pt-fancyzones-settings.png)

### Choose your layout (Layout Editor)

> When first launched, the zones editor presents a list of layouts that can be adjusted by how many windows are on the monitor. Selecting a layout shows a preview of that layout on the screen. The selected layout is applied automatically. Double-clicking a layout will apply it and close the editor.

Reviewer: ```suggestion
When you first launch the zones editor, you'll see a list of layouts that can be adjusted by how many windows are on the monitor. Selecting a layout shows a preview of that layout on the screen. The selected layout is applied automatically. Double-clicking a layout will apply it and close the editor.
```

Simplified

### Space around zones

> The **Show space around zones** setting determines what sort of margin will surround each FancyZone window. Enter a custom value for the width of the margin in **Space around zones**. With the Zones Editor open, change the **Show space around zones** after changing the values to see the new value applied.

Reviewer: ```suggestion
**Show space around zones** sets the size of margin around each FancyZone window. Enter a custom value for the width of the margin in **Space around zones**. With the zones editor open, change **Show space around zones** after changing the values to see the new value applied.
```

> The **Distance to highlight adjacent zones** sets a custom value for the amount of space between zones until they snap together, or before both are highlighted enabling them to merge together.

Reviewer: ```suggestion
**Distance to highlight adjacent zones** sets a custom value for the amount of space between zones until they snap together, or before both are highlighted enabling them to merge together.
```

> The **Default layout for horizontal monitor orientation** and **Default layout for vertical monitor orientation** flags can be used to set which layout will be used as default when display configuration is changed in the system (e.g. new display was added).

Reviewer: ```suggestion
**Default layout for horizontal monitor orientation** and **Default layout for vertical monitor orientation** set which layout to use as the default when the display configuration is changed in the system (for example, if you add a new display).
```

Simplified and removed the `e.g.` abbreviation.
